tests/Unit/DepreciationTest.php

```diff
@@ -62,8 +63,9 @@ public function testDepreciationPercentage()
             ->laptopMbp()
             ->create(
                 [
-                    'category_id' => Category::factory()->assetLaptopCategory()->create(),
-                    'purchase_date' => now()->subDecade(),
+                    //not sure how this ever worked...
+                    //'category_id' => Category::factory()->assetLaptopCategory()->create(),
```

Comment:
I think we can remove this comment. I think turning on validation exceptions is causing this?